README.md

```diff
@@ -73,16 +70,19 @@ make install
 Run `mongocli help` for a list of available commands
 or check our [online documentation](https://docs.mongodb.com/mongocli/master/) for more details.
 
-### Getting API Keys
+### Configuring MongoCLI with Atlas
+To use `mongocli` with Atlas you'll need to run `mongocli auth login` and follow the prompted steps on your terminal.
+
+### Configuring MongoCLI with Ops Manager and Cloud Manager
 
-To use `mongocli` you'll need to get API keys, to get them follow the documentation 
-appropriate for the service you're using, 
+#### Getting API Keys (Ops manager / Cloud Manager)
+To use `mongocli` you'll need to get API keys, to get them follow the documentation
+appropriate for the service you're using,
```

Comment:
To use `mongocli`, create API keys. To learn more, see the documentation for the service you're using: